po/zh-TW.po

```diff
@@ -9275,23 +9280,28 @@ msgid ""
 "```"
 msgstr ""
 
-#: src/error-handling/panic-unwind.md:19
+#: src/error-handling/panic-unwind.md:21
```

Comment:
> Looks like a rebase mistake?

Probably not: the line numbers have likely shifted since the `messages.pot` file were generated. The line numbers will be updated the next thing you run `msgmerge --update` on the file. They are only used for you to jump back to the source file, so it's harmless if they become outdated.